### PR TITLE
94755-item-browser-subnav-visual-treatment

### PR DESCRIPTION
**Pull request by joebri**
94755
### Review
 - [x] @boskya

### After Merge

- [ ] [site](https://daptiv.github.io/style-guide) is updated

### CODE REVIEW

**Comment by boskya** [comment]
@jtrinklein

File: styles/nav.scss [hunk 1]
```diff
@@ -153,6 +153,32 @@ $daptiv-subnav-tab-h-pad: 14px;
...
+        @extend .daptiv-subnav-tab;
+
+        &::before {
+            @extend .fa;
+            content: $fa-var-angle-double-left;
+            font-size: $icon-size-small;
+            margin-right: 3px;
+            position: relative;
+            top: 1px;
+        }
```
Comment (jtrinklein) [inline styles/nav.scss]: these should be in a new section external to navigation: Iconography
ideally something like this
```sass
$mixin icon-small($position, $content) {
    &::$position {
        font-size: $icon-size-small;
        margin-right: 3px; /* probably create a variable for this */
        position: relative;
        top: 1px; /* probably create a variable for this  */
    }
}
.icon {
    @extend .fa;
    
    &.previous {
        =icon-small(before, $fa-var-angle-double-left);
    }
    &.next {
        =icon-small(after, $fa-var-angle-double-right);
    }
}
```

so then you would use it like this

```html
<a href="#prev" class="daptiv-subnav-tab icon previous">Previous</a>
<a href="#next" class="daptiv-subnav-tab icon next">Next</a>
```

Comment (boskya) [inline styles/nav.scss]: :+1:

Comment (jtrinklein) [inline styles/nav.scss]: agreed, but would only be nice to have for this pr. I can catch it on the semantic revision story otherwise.

**Comment by boskya** [comment]
merge conflicts but otherwise good to go.

**Pull request merged**